chapters/project-guide/project-guide.rst

```diff
@@ -123,26 +120,34 @@ Workspace
 
 ``/workspace``
 
-This :term:`workspace is a special folder. You will **not** create this folder yourself. You should design your **app.py** in a way that when you run it for the first, this folder is created during the :term:`indexing`. And this workspace folder should be created under the main directory.
+This :term:`workspace` is a special folder. You will **not** create this folder yourself. You should design your **app.py** in a way that when you run it for the first time, this folder is created during the :term:`indexing`.
 
 Dockerfile
 ---------
+`
+``/Dockerfile``
 
-``/app.py``
 
+This is another optional element, but if you want to have one you can have one under the main directory
 
-This is another optional element, but if you want to have a Docker image you can have one under the main directory
-
-README
+.dockerignore
 ---------
 
-``/README.md``
+``/.dockerignore``
 
-Finally, we have our README. It is good practice to have this for you to show all the necessary steps you'll need to do to run your app. And we have this living under the main folder too.
+Don't forget to add here all the files that you don't want to include in your initial build context. The Docker daemon will skip those files for the **docker build**
 
 
+.gitignore
+---------
 
+``/.gitignore``
 
+Add here whatever files you don't want to commit. The **.gitignore** is usually placed under the main directory and we follow that convention in Jina.
```

Comment:
`A list of files you don't want git to commit`

I think we can cut the last sentence. It's implied in the file path above